e2e/protractor/suites/actions/library/library-actions.test.ts

```diff
@@ -1,429 +0,0 @@
-      await adminApiActions.sites.addSiteMember(siteSearchPublic2Admin, username, SITE_ROLES.SITE_CONTRIBUTOR.ROLE);
-      await adminApiActions.sites.addSiteMember(sitePublic4Admin, username, SITE_ROLES.SITE_MANAGER.ROLE);
-      await adminApiActions.sites.addSiteMember(sitePublic5Admin, username, SITE_ROLES.SITE_MANAGER.ROLE);
-      await apis.user.queries.waitForSites(siteSearchPublic2Admin, { expect: 1 });
-    });
-
-    afterAll(async () => {
-      await adminApiActions.deleteSites(adminSites);
-      await apis.user.sites.deleteSites(userSites);
-    });
-
-    it('[C290106] from My Libraries', async () => {
-      await page.goToMyLibrariesAndWait();
-      await dataTable.selectItem(sitePublic2Admin);
-      await BrowserActions.click(toolbar.leaveButton);
-      await page.waitForDialog();
-      await BrowserActions.click(confirmDialog.okButton);
-
-      expect(await page.getSnackBarMessage()).toEqual(`You have left the library`);
-      expect(await dataTable.isItemPresent(sitePublic2Admin)).toBe(false, `${sitePublic2Admin} is displayed`);
-    });
-
-    it('[C290110] from Favorite Libraries', async () => {
-      await page.goToFavoriteLibrariesAndWait();
-      await dataTable.selectItem(sitePublic3Admin);
-      await BrowserActions.click(toolbar.leaveButton);
-      await page.waitForDialog();
-      await BrowserActions.click(confirmDialog.okButton);
-
-      expect(await page.getSnackBarMessage()).toEqual(`You have left the library`);
-      expect(await dataTable.isItemPresent(sitePublic3Admin)).toBe(true, `${sitePublic3Admin} is not displayed`);
-    });
-
-    it('[C306961] from Search Results', async () => {
-      await toolbar.clickSearchIconButton();
-      await searchInput.clickSearchButton();
-      await searchInput.checkLibraries();
-      await searchInput.searchForLibrary(siteSearchPublic2Admin);
-      await dataTable.waitForBody();
-
-      await dataTable.selectItem(siteSearchPublic2Admin);
-      await BrowserActions.click(toolbar.leaveButton);
-      await page.waitForDialog();
-      await BrowserActions.click(confirmDialog.okButton);
-
-      expect(await page.getSnackBarMessage()).toEqual(`You have left the library`);
-      expect(await dataTable.isItemPresent(siteSearchPublic2Admin)).toBe(true, `${siteSearchPublic2Admin} is not displayed`);
-    });
-
-    it('[C290136] Confirmation dialog UI', async () => {
-      await page.goToMyLibrariesAndWait();
-      await dataTable.selectItem(sitePublic4Admin);
-      await BrowserActions.click(toolbar.leaveButton);
-      await page.waitForDialog();
-
-      expect(await confirmDialog.isDialogOpen()).toBe(true, 'Confirm delete dialog not open');
-      expect(await confirmDialog.getDialogTitle()).toContain('Leave this library?');
-      expect(await confirmDialog.getText()).toContain('Leaving will remove your access.');
-      expect(await confirmDialog.isOkEnabled()).toBe(true, 'OK button is not enabled');
```

Comment:
isOkEnabled function is unused now so please remove it

Reply:
deleted in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)